R/Data-class.R

```diff
@@ -82,7 +82,7 @@ NULL
 #'
 #' @note `ID` and `cohort` can be missing, then a warning
 #'   will be issued and the variables will be filled with default
-#'   IDs and best guesses, respectively.
+#'   IDs and best guesses cohort from `\{1, 2, 3, ...\}`.
```

Comment:
```suggestion
#'   values `1, 2, 3, ...`.
```

Reply:
But the cohort can be like `1,1,1,2,2,3,3,3` which is not "values `1, 2, 3, ...`".